src/main/java/Event.java

```diff
@@ -0,0 +1,32 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Event task.
+ */
+public class Event extends Task{
+    public LocalDate time;
+    public Event(String description, LocalDate time) {
+        super(description);
+        this.time = time;
+    }
+
+    public Event(boolean isDone, String description, LocalDate due) {
+        super(isDone, description,'E');
+        this.time = due;
+        String unparseMessage = "E";
+        if (isDone) {
+            unparseMessage += "1";
+        } else {
+            unparseMessage += "0";
+        }
+        unparseMessage += description;
+        unparseMessage += ",";
+        unparseMessage += due;
```

Comment:
This code seems to be repeated in a few parts, would it be better to have your `Parser` class handle some of these parsing?

Reply:
Thank you for your suggestion!